frontend/src/Theme.css

```diff
@@ -118,6 +121,12 @@
 	--tooltip-background-colour: #fff;
 	--tooltip-border-colour: #000;
 	--tooltip-text-colour: #000;
+
+	/* themed buttons */
+	--themed-button-hover-colour: #11484b;
+
+	/* Defence Mechanism */
+	--summary-hover-colour: #000;
```

Comment:
I'm going to suggest removing this one, and its usage in DefenceMechanism.css, because we have another issue (#723) to adjust styling of these detail elements in the sidebar.